test/k8sT/manifests/default-deny-egress.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: default-deny
```

Comment:
Please name `default-deny-egress` to make it obvious in CI logs when it fails

Reply:
Done